Backend/load-test/README.md

```diff
@@ -1,26 +1,23 @@
 ### To set up your environment for load-test
 
 ```
-cd ./Backend/load-test
-bash setup.sh
-source ./bin/activate
-pip3 install -r requirements.txt
+You should have the Nammayatri servers and services running using
+, run-svc -d
+, run-mobility-stack-dev
 ```
 
 ### To run the load-test
 
+You can run the load test by
 ```
-, run-load-test
+ , run-load-test-nix
```

Comment:
Is this correct?

`, run-load-test-nix` just runs `nix run` which fails for me:

I can run the load-test on my machine:

<img width="635" alt="image" src="https://github.com/nammayatri/nammayatri/assets/3998/204a162b-77b5-4422-8bbd-2244f1370e4d">